File: 04-CSHARP/06-Arreglos/02introducirValoresArray.cs
// Introducir valores a un array en C#

// Crear el array
int largoArray;
do
{
    Console.WriteLine("Ingresa el largo del arreglo");
    largoArray = Convert.ToInt32(Console.ReadLine());
} while (largoArray <= 0);

int[] numeros = new int[largoArray];

// Solicitar valores al usuario
for (int i = 0; i < numeros.Length; i++)
{
    Console.WriteLine("Ingresa un valor");
    numeros[i] = Convert.ToInt32(Console.ReadLine());
}

// Imprimir los valores del array 
foreach (int numero in numeros)
{
    Console.WriteLine($"Valor: {numero}");
}

File: 03_CPP/06_arreglos/02introducirValoresArray.cpp
// Introducir valores a un array en C++
#include <iostream>
using namespace std;

int main()
{
    int largoArray;
    do
    {
        cout << "Ingresa el largo del arreglo: ";
        cin >> largoArray;
    } while (largoArray <= 0);
    cout << "Largo del array: " << largoArray << endl;
    cout << endl;

    // int numeros[largoArray];
    int *numeros = new int[largoArray];

    // Solicitar valores al usuario
    for (int i = 0; i < largoArray; i++)
    {
        cout << "Ingresa el valor " << i << endl;
        cin >> numeros[i];
    }

    // Iterar todos los numeros
    for (int i = 0; i < largoArray; i++)
    {
        cout << "Valor " << i << ": " << numeros[i] << endl;
    }

    return 0;
}